### ISSUE TITLE
Cannot import bqplot_gl when using bqplot 0.13

### ISSUE THREAD

**Issue by pllim** [glue-viz/glue-jupyter#377]
Example log: https://github.com/spacetelescope/jdaviz/actions/runs/5724772570/job/15511946760

```
    from glue_jupyter.bqplot.image import BqplotImageView
glue_jupyter/bqplot/image/__init__.py:2: in <module>
    from .viewer import *  # noqa
glue_jupyter/bqplot/image/viewer.py:7: in <module>
    from ..scatter.layer_artist import BqplotScatterLayerArtist
glue_jupyter/bqplot/scatter/__init__.py:1: in <module>
    from .layer_artist import *  # noqa
glue_jupyter/bqplot/scatter/layer_artist.py:3: in <module>
    from ..compatibility import ScatterGL, ImageGL
glue_jupyter/bqplot/compatibility.py:8: in <module>
    import bqplot_gl
E   ModuleNotFoundError: No module named 'bqplot_gl'
```

I think you need a conditional install of bqplot-gl when `bqplot>0.12` here:

https://github.com/glue-viz/glue-jupyter/blob/c848e4f133a9843081e132de9adf96b2239070ed/setup.cfg#L16

**Comment by dhomeier** [comment]
Discussed in #302 – but at this point, trying to install `bqplot-gl` still falls back on 0.12.x; probably need to wait for a stable 0.13 release ot finalise this (and/or pin to `bqplot<0.13.0a0` until then).

**Comment by dhomeier** [comment]
Actually, since bqplot-gl 0.0.0 is already available as a placeholder for `bqplot<0.13,>=0.12`, there is nothing keeping us from adding it to `install_requires` right now (could even have only bqplot-gl in there, if it weren't for the minimum version of bqplot).

**Comment by dhomeier** [comment]
Resolved by #385
